### PR TITLE
add formatMsg in textProcessor, & classify text procedure in comment.…

### PR DESCRIPTION
**Pull request by Crysitna**
… & fix SQ bug

### CODE REVIEW

File: sample-spring-boot-kitchensink/src/main/java/com/example/bot/spring/TextProcessor.java [hunk 1]
```diff
@@ -103,23 +108,70 @@ private String classifyText(String userId, String text){
...
+		}
+	}
+	
+	/** truncate all non-digit and non-char elements from user input (decimal point is reserved)
+	 * and transform the input into lower case; 
+	 * @param message: user-input;
+	 * 
+	 * NOTICE: it's possible for formatMsg() to return a empty text, which need to be handled in its calling function
+	 */
+	private String formatMsg(String message) {
```
Comment (zysjimmy) [inline sample-spring-boot-kitchensink/src/main/java/com/example/bot/spring/TextProcessor.java]: What is the 'formatMsg' function for?

Comment (Crysitna) [reply sample-spring-boot-kitchensink/src/main/java/com/example/bot/spring/TextProcessor.java]: some punctuation cannot be recognized by database/java

Comment (Crysitna) [reply sample-spring-boot-kitchensink/src/main/java/com/example/bot/spring/TextProcessor.java]: and sometimes user just sent us strange symbol =)

File: sample-spring-boot-kitchensink/src/test/java/com/example/bot/spring/KitchenSinkTester.java [hunk 2]
```diff
@@ -50,9 +51,9 @@ public void testUQ() throws Exception {
 		assertThat(result).isEqualTo("Sorry, I can't answer your question. My colleague will follow up with you.");
 	}
 
-  /*
+  
   // only applicable when textProcessor calling no external function
-	@Test
```
[the remark this answers is not in the document]
Comment (Crysitna) [reply sample-spring-boot-kitchensink/src/test/java/com/example/bot/spring/KitchenSinkTester.java]: this is what we learn from today's lecture 💯

**Pull request merged**